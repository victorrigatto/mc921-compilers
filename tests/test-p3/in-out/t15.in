int main() {
    char y = 'a';
    return y;
}
