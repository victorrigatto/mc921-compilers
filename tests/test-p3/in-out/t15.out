SemanticError: Return of type(char) is incompatible with type(int) function definition @ 3:5
